_posts/2015-02-18-ApiBlueprint-button.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Social blueprint
+excerpt: Share your blueprint with others and learn from masters.
+layout: post
+date: 2015-02-18 00:00:00 +0100
+author: vincenzo
+published: true
+commets: true
+---
+
+Recently, we have received a lot of feedback from users that would like to learn more about API Blueprint and how to present stunning and beautiful documentation for their users.
+
+We care about your feedback and we think that documentation and tutorials about [API Blueprint](http://apiblueprint.org) are extremely important. Given that we're hard working on this, and that tons of stuff is coming, today I am glad to announce that we've made another step forward in that direction.
+
+One of the most requested feature was the ability to look to the other blueprints as a source of inspiration from that, and today this is possible, according to your blueprint sharing policies.
+
+Whenever you're surfing a public blueprint, a new **API Blueprint** button will appear on header bar. Clicking on that will open up a new window with raw source code, that can be easily grabbed, studied, modified and republished as well.
```

Comment:
This is awesome, does this link to the GitHub source so people can fork & pull request to make changes directly in the case of public linked GitHub repositories?